projects/core/src/global-message/http-interceptors/handlers/http-error.handler.ts

```diff
@@ -49,8 +51,15 @@ export abstract class HttpErrorHandler implements Applicable {
    *
    * Example: 'User is disabled' will be transformed to 'user_is_disabled'.
    */
-  createErrorTranslationKey(reason: string): string {
-    return reason.toLowerCase().replace(/ /g, '_');
+  getErrorTranslationKey(reason: string): string {
+    const translationPrefix = `httpHandlers.badRequest`;
+
+    // TODO: Remove feature flag `isLevel` check for 6.0 (CXSPA-2413)
+    if (this.featureConfigService?.isLevel('5.2') || !reason) {
```

Comment:
This will always return the `PleaseLoginAgain` for feature 5.2+ right? Is the logic backwards?

Reply:
I changed the if statement to affect versions below 5.2.